test/pass/simple-build-id.cmake

```diff
@@ -0,0 +1,23 @@
+################################################################################
+# Copyright (C) 2017 Advanced Micro Devices, Inc.
+################################################################################
+
+find_program(GIT NAMES git)
+
+set(ENV{CPACK_DEBIAN_PACKAGE_RELEASE} "dnnhc.3998")
+
+# Try to test without git
+file(MAKE_DIRECTORY ${TMP_DIR}/repo)
+execute_process(COMMAND ${GIT} describe --dirty --long --always WORKING_DIRECTORY ${TMP_DIR}/repo OUTPUT_VARIABLE GIT_TAG RESULT_VARIABLE RESULT)
+if(NOT ${RESULT} EQUAL 0)
+    set(GIT_TAG 0)
+endif()
+write_version_cmake(${TMP_DIR}/repo 1.0 "
+    test_expect_eq(\${CPACK_DEBIAN_PACKAGE_RELEASE} $ENV{CPACK_DEBIAN_PACKAGE_RELEASE})
```

Comment:
This should check the project versions. Major should be 1, minor 0, patch 0, and tweak dnnhc.3998.

Reply:
Updated as discussed